locales/gl-ES.json

```diff
@@ -0,0 +1,537 @@
+    "notifications": "Notificacións",
+    "profile": "Perfil",
+    "search": "Buscar",
+    "select_feature_flags": "Toggle Feature Flags",
+    "select_font_size": "Tamaño do texto",
+    "select_language": "Idioma da interface",
+    "settings": "Axustes",
+    "show_intro": "Mostrar intro",
+    "toggle_theme": "Mudar decorado",
+    "zen_mode": "Modo Cen"
+  },
+  "notification": {
+    "favourited_post": "favoreceu a túa publicación",
+    "followed_you": "seguiute",
+    "followed_you_count": "{0} persoas seguíronte|{0} persoa seguiute|{0} persoas seguíronte",
+    "missing_type": "FALTA o notification.type:",
+    "reblogged_post": "promoveu a túa publicación",
+    "request_to_follow": "solicitou seguirte",
+    "signed_up": "creou a conta",
+    "update_status": "actualizou a publicación"
+  },
+  "placeholder": {
+    "content_warning": "Escribe aquí o aviso",
+    "default_1": "En que estás a pensar?",
+    "reply_to_account": "Responder a {0}",
+    "replying": "Respondendo",
+    "the_thread": "a conversa"
+  },
+  "pwa": {
+    "dismiss": "Desbotar",
+    "title": "Dispoñible nova versión de Elk!",
+    "update": "Actualizar",
+    "update_available_short": "Actualizar Elk",
+    "webmanifest": {
+      "canary": {
+        "description": "Un cliente áxil para Mastodon (canary)",
+        "name": "Elk (canary)",
+        "short_name": "Elk (canary)"
+      },
+      "dev": {
+        "description": "Un cliente áxil para Mastodon (dev)",
+        "name": "Elk (dev)",
+        "short_name": "Elk (dev)"
+      },
+      "preview": {
+        "description": "Un cliente áxil para Mastodon (preview)",
+        "name": "Elk (preview)",
+        "short_name": "Elk (preview)"
+      },
+      "release": {
+        "description": "Un cliente áxil para Mastodon",
+        "name": "Elk",
+        "short_name": "Elk"
+      }
+    }
+  },
+  "search": {
+    "search_desc": "Buscar persoas e cancelos",
+    "search_empty": "Non atopamos nada con eses termos de busca"
+  },
+  "settings": {
+    "about": {
+      "label": "Acerca de",
+      "meet_the_team": "Coñece ao equipo",
+      "sponsor_action": "Patrocínanos",
+      "sponsor_action_desc": "Para axudar ao equipo que desenvolve Elk",
+      "sponsors": "Patrocinios",
+      "sponsors_body_1": "Elk é posible grazas ao xeneroso patrocinio e axuda de:",
+      "sponsors_body_2": "Todas as empresas e persoas que patrocinan a Team Elk e colaboradoras.",
+      "sponsors_body_3": "Se estás desfrutando da app, considera patrocinarnos:",
+      "version": "Versión"
+    },
+    "account_settings": {
+      "description": "Edita a configuración da túa conta na interface de Mastodon",
+      "label": "Axustes da conta"
+    },
+    "interface": {
+      "color_mode": "Tono da cor",
+      "dark_mode": "Escuro",
+      "default": " (por defecto)",
+      "font_size": "Tamaño da letra",
+      "label": "Interface",
+      "light_mode": "Claro",
+      "size_label": {
```

Comment:
this entry seems to be removed, can you review it?

Reply:
:thinking:  oh!, it has changed since I had downloaded it (two days ago). I'm fixing it now and also add galician entry in i18n.ts. Please excuse me. Thanks.


Ahora he lo arreglo y también el  i18n.ts, veo que no están por orden alfabético, así que lo pondré abajo del todo. Disculpad el error, gracias.